### DISCUSSION TITLE
Problems with authentication.

### DISCUSSION THREAD

**Discussion by marcusio888** [dragonflydb/dragonfly#3110] [General]
I have Dragonfly installed in a cluster via helmchart, I activated the password part in the values.yaml but I cannot authenticate.

dfly@dragonfly-0:/data$ env | grep -i pass
DFLY_requirepass=XhDx24Ab0LV1qW7FhQqYygdTBKyXwb

dfly@dragonfly-0:/data$ redis-cli
127.0.0.1:6379> keys *
(error) NOAUTH Authentication required.
127.0.0.1:6379> auth default XhDx24Ab0LV1qW7FhQqYygdTBKyXwb
(error) WRONGPASS invalid username-password pair or user is disabled.

How is this issue resolved?

**Comment by romange** [comment]
Seems to be working 🤷🏼 . Running locally:
```
docker run  -p 6379:6379 --rm -e DFLY_requirepass=foo docker.dragonflydb.io/dragonflydb/dragonfly
```
```
> redis-cli -a foo
127.0.0.1:6379> ping
PONG
127.0.0.1:6379> 
```

**Comment by marcusio888** [comment]
I was already able to solve the problem with the secret, I was using lens and it appears that it saves a space character which cannot be seen in the variable.

**Reply by romange** [reply]
lens? what do you mean?

**Reply by marcusio888** [reply]
Yes openLens, when managing the secret with openlens you can see that it added a space character. I solved everything via command so thanks.
